lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,283 @@
+	t.end();
+});
+
+tape( 'the function solves hard problems', function test( t ) {
+	var bits;
+	var q;
+
+	// Test case #1
+	q = cdiv( 1.0, 1.0, 1.0, pow( 2.0, 1023.0 ) );
+
+	bits = bitsCorrect( q[0], pow( 2.0, -1023.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], -pow( 2.0, -1023.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #2
+	q = cdiv( 1.0, 1.0, pow( 2.0, -1023.0 ), pow( 2.0, -1023.0 ) );
+
+	bits = bitsCorrect( q[0], pow( 2.0, 1023.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], 0.0 );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #3
+	q = cdiv( pow( 2.0, 1023.0 ), pow( 2.0, -1023.0 ),
+		pow( 2.0, 677.0 ), pow( 2.0, -677.0 ) );
+
+	bits = bitsCorrect( q[0], pow( 2.0, 346.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], -pow( 2.0, -1008.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #4
+	q = cdiv( pow( 2.0, 1023.0 ), pow( 2.0, 1023.0 ), 1.0, 1.0 );
+
+	bits = bitsCorrect( q[0], pow( 2.0, 1023.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], 0.0 );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #5
+	q = cdiv( pow( 2.0, 1020.0 ), pow( 2.0, -844.0 ),
+		pow( 2.0, 656.0 ), pow( 2.0, -780.0 ) );
+
+	bits = bitsCorrect( q[0], pow( 2.0, 364.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], -pow( 2.0, -1072.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #6
+	q = cdiv( pow( 2.0, -71.0 ), pow( 2.0, 1021.0 ),
+		pow( 2.0, 1001.0 ), pow( 2.0, -323.0 ) );
+
+	bits = bitsCorrect( q[0], pow( 2.0, -1072.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], pow( 2.0, 20.0 ) );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #7
+	q = cdiv( pow( 2.0, -347.0 ), pow( 2.0, -54.0 ),
+		pow( 2.0, -1037.0 ), pow( 2.0, -1058.0 ) );
+
+	bits = bitsCorrect( q[0], 3.898125604559113300e289 );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+	bits = bitsCorrect( q[1], 8.174961907852353577e295 );
+	t.deepEqual( bits, [1, 11, 52], 'Sign, exponent, and mantissa bits are correct' );
+
+	// Test case #8
+	q = cdiv( pow( 2.0, -1074.0 ), pow( 2.0, -1074.0 ),
+		pow( 2.0, -1073.0 ), pow( 2.0, -1074.0 ) );
+
+	bits = bitsCorrect( q[0], 0.6 );
+	t.deepEqual( bits, [1, 11, 49], 'Sign, exponent, and mantissa bits are correct' );
```

Comment:
This was the only test that didn't get all bits correct. It had 49 / 52 correct where I expected perhaps one or two more. I checked it and checked again. Not sure.